.github/workflows/PublishIncrementalNuget.yml

```
@@ -0,0 +1,49 @@
+        with:
+          files: |
+            "**/Consolonia.csproj"
+          version: bump-build
+  
+      - name: Restore dependencies
+        run: dotnet restore source/Consolonia.sln
+
+      - name: Build
+        run: dotnet build source/Consolonia.sln -c Release --no-restore 
+  
+      - name: dotnet pack 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)